doc/content/getting-started/cli/installing-cli/_index.md

```diff
@@ -61,6 +61,15 @@ For a standard deployment on `thethings.example.com`, all you need is:
 $ ttn-lw-cli use thethings.example.com [--fetch-ca] [--user] [--overwrite]
 ```
 
+{{< warning >}} If using an `au1` or `nam1` cluster, you must manually update the Identity Server endpoints, as there is no `au1` or `nam1` Identity Server. Use the `eu1` endpoints by setting the following in your configuration file:
```

Comment:
This is all going to be very confusing. Most of this page describes `thethings.example.com` and now we're suddenly starting about specific deployments.

There's absolutely no requirement for (private) deployments to call their Identity Server `eu1`, (private) deployments also don't have to follow the `au1`/`nam1` naming. 

Also, it's going to be very annoying if we have to update this page every time we add clusters to either The Things Network or The Things Stack Cloud, and it's going to be very confusing if we'll have a The Things Stack Cloud cluster name that doesn't exist in The Things Network or vice versa.

So I actually think this doesn't belong on this page.